gradle/utils.gradle

```diff
@@ -74,14 +74,17 @@ ext {
     }
 
     exePresent = { exe ->
-        ["where $exe", "which $exe"].any {
-            try {
-                def findExe = it.execute()
-                findExe.waitForProcessOutput()
-                return findExe.exitValue() == 0
-            } catch (any) {
-                return false
-            }
+        def osName = System.getProperty('os.name').toLowerCase()
+        if (osName.contains('windows')) {
+            providers.exec {
+                ignoreExitValue = true
+                commandLine('where', exe)
+            }.result.get() == 0
+        } else {
+            providers.exec {
+                ignoreExitValue = true
+                commandLine('which', exe)
+            }.result.get() == 0
         }
```

Comment:
A change here may change a behavior of  `onlyIf` block that test existence of executables.
It was not recommended for configuration-cache enabled build, so I changed to use recommended way that use `providers.exec` gradle command.